<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HDC2-L-33245XC" failures="1" tests="1" name="Vitrai.UpdateQuestionnaire" time="1.027" errors="0" timestamp="2025-06-06T14:44:02 IST" skipped="0">
  <testcase classname="Vitrai.UpdateQuestionnaire" name="updatequestionnaire" time="1.027">
    <failure message="Initial POST request failed. Status Code: 400, Response: {&quot;type&quot;:&quot;https://tools.ietf.org/html/rfc9110#section-15.5.1&quot;,&quot;title&quot;:&quot;One or more validation errors occurred.&quot;,&quot;status&quot;:400,&quot;errors&quot;:{&quot;LinkId&quot;:[&quot;The LinkId field is required.&quot;],&quot;QuestionnaireResponseId&quot;:[&quot;The QuestionnaireResponseId field is required.&quot;]},&quot;traceId&quot;:&quot;00-4a9f1c302ab7d55d8176a580e0ca2841-3c060ed5af4d9a1d-00&quot;}" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Initial POST request failed. Status Code: 400, Response: {"type":"https://tools.ietf.org/html/rfc9110#section-15.5.1","title":"One or more validation errors occurred.","status":400,"errors":{"LinkId":["The LinkId field is required."],"QuestionnaireResponseId":["The QuestionnaireResponseId field is required."]},"traceId":"00-4a9f1c302ab7d55d8176a580e0ca2841-3c060ed5af4d9a1d-00"}
at org.testng.Assert.fail(Assert.java:111)
at Vitrai.UpdateQuestionnaire.updatequestionnaire(UpdateQuestionnaire.java:43)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- updatequestionnaire -->
</testsuite> <!-- Vitrai.UpdateQuestionnaire -->
